### PR TITLE
Midi 16/tokenization review dashboard

### PR DESCRIPTION
**Pull request by WojciechMat**
[MIDI-16: tokenization review dashboard](https://sacrebleu.atlassian.net/jira/software/projects/MIDI/boards/7?selectedIssue=MIDI-16)

I changed the dataset to include unprocessed samples.

### CODE REVIEW

**Comment by WojciechMat** [comment]
## Dashboard
You can now run tokenization review dashboard with 
```
streamlit run --server.port 4466 dashboard.py
```
first column is a real sample, second one is quantized.

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,4 @@
 .idea/*
-*__pycache__*
\ No newline at end of file
+*__pycache__*
+dataset*.pt
+tmp/*
```
Comment (roszcz) [inline .gitignore]: You still should add this folder to the repo :)

**Review by roszcz — COMMENTED** [review]
Looks cool! Can you show more information on the dashboard? Specifically I mean column headers, and some details about quantization method used

File: .gitignore (hunk 1 above)
Comment (WojciechMat) [reply .gitignore]: sry, I forgot to force add .keep

**Comment by WojciechMat** [comment]
### Choose tokenization method
You can now choose number of bins to quantize samples to view.
image

**Review by roszcz — APPROVED** [review]

**Pull request merged**